Updating Barefoot logo and website

rkeverman opened the pull request.

stolsma commented:
@rkeverman You referenced to a non existed barefoot image... I suppose you wanted to reference the also added new Intel logo??

rkeverman commented:
I'm not sure if I'm able to reply to this message, but I inadvertently included the wrong source link from assets for the Barefoot logo. But I corrected it and created a new PR. I also changed out the asset link for the Intel logo because there is a newly branded version.

While I wait for my permissions to update with ONF, would you be able to merge these PRs to the master so the webpage is appropriately updated?

-Rachel
________________________________
From: Sander Tolsma <notifications@github.com>
Sent: Wednesday, November 18, 2020 12:45 AM
To: p4lang/p4lang.github.io <p4lang.github.io@noreply.github.com>
Cc: Rachel Everman <rachel.everman@barefootnetworks.com>; Mention <mention@noreply.github.com>
Subject: Re: [p4lang/p4lang.github.io] Updating Barefoot logo and website (#619)


@rkeverman<https://github.com/rkeverman> You referenced to a non existed barefoot image... I suppose you wanted to reference the also added new Intel logo??

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/p4lang/p4lang.github.io/pull/619#issuecomment-729528866>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AE3F6NP5DO6MO22F3DCVR2LSQOCTBANCNFSM4TZJXHRA>.

jnfoster commented:
The CLA is only advisory at the moment. So you can and have merged :-)
